pennylane/qinfo/transforms.py

```diff
@@ -385,10 +385,10 @@ def circ(params):
 
         params = pnp.random.random(2)
 
-    >>> print(qml.qinfo.classical_fisher(circ)(params))
-    (tensor([[0.13340679, 0.03650311],
-             [0.03650311, 0.00998807]], requires_grad=True)
-    >>> print(qml.jacobian(qml.qinfo.classical_fisher(circ))(params))
+    >>> qml.qinfo.classical_fisher(circ)(params)
+    tensor([[0.28096197, 0.36228429],
```

Comment:
Outputs simply changed because of random outputs. `print` doesn't show the type of the output, hence the removed `print` calls.